core/local/change.js

```diff
@@ -432,22 +434,28 @@ function includeAddEventInFileMove (sameInodeChange /*: ?LocalChange */, e /*: L
   return true
 }
 
-function includeAddDirEventInDirMove (sameInodeChange /*: ?LocalChange */, e /*: LocalDirAdded */) {
+// This is based on a bug in chokidar where
+// When a moved directory overwrites its destination on macOS with APFS,
```

Comment:
nit: it's a bit surprising to find an uppercase letter in the middle of a sentence. It makes it a bit harder to read.